app/src/components/Shared/MenuMobile.jsx

```diff
@@ -14,22 +14,30 @@ class MenuMobile extends Component {
     let userLinks;
     if (this.props.loggedUser) {
       userLinks = (
-        <li>
-          <a>{this.props.loggedUser.displayName}</a>
-          <ul>
-            <li>
-              <a onClick={this.logout}>
-                Logout
-              </a>
-            </li>
-          </ul>
-        </li>
+        <div>
+          <span
```

Comment:
It would probably be better to let an "a" element because it is more accessible (focusable).